### ISSUE TITLE
Cabelos

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments-new#150]
### Post de Referência:

https://www.vish.blog.br/imagens/cabelos/

**Comment by maryon-telvanni** [comment]
Se uma mulher disse, então deve ser verdade

**Comment by angellisy** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/cabelos/#comment_1187606501>):

> Se uma mulher disse, então deve ser verdade

E como você pode ter certeza de que sou mulher?

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/cabelos/#comment_1187609183>):

> E como você pode ter certeza de que sou mulher?

Verdade. Ela pode ser um cara de 130kg que vive no porão da mãe no subúrbio de Detroit chamado Chuck :stuck\_out\_tongue:

**Comment by maryon-telvanni** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/cabelos/#comment_1187609183>):

> E como você pode ter certeza de que sou mulher?

![null](<https://i.imgur.com/ZaHY4gW.png>)

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/cabelos/#comment_1187636241>):

> Verdade. Ela pode ser um cara de 130kg que vive no porão da mãe no subúrbio de Detroit chamado Chuck 😛

![null](<https://i.imgur.com/wZMKtUb.png>)
